torchdata/datapipes/iter/util/converter.py

```diff
@@ -26,13 +26,22 @@ class IterToMapConverterMapDataPipe(MapDataPipe):
     with exactly two objects. The first object of each item becomes a key in
     the new dictionary, and the second object the corresponding value.
 
+    For the opposite converter, use :class:`.MapToIterConverter`.
```

Comment:
Wow. Have you checked this is gonna work?

Reply:
I did and it does!

Reply:
Thank you for verification!

Reply:
![Screen Shot 2022-05-03 at 2 17 23 PM](https://user-images.githubusercontent.com/4935152/166517004-f3c5b620-7ff5-4d0d-a750-89358386ab71.png)